libs/api_common/windows_platform_common.cpp

```diff
@@ -44,6 +44,18 @@ const ebpf_context_descriptor_t g_bind_context_descriptor = {
 const EbpfProgramType windows_bind_program_type =
     PTYPE("bind", &g_bind_context_descriptor, (uint64_t)&EBPF_PROGRAM_TYPE_BIND, {"bind"});
 
+const ebpf_context_descriptor_t g_flow_context_descriptor = {
+    sizeof(flow_md_t), 0, EBPF_OFFSET_OF(flow_md_t, app_id) + sizeof(uint64_t), -1};
```

Comment:
why the offset of app_id? Same question for packet_length.

Reply:
Since it is the last variable of the struct, I assumed that its offset + its size would be the offset of the pointer to end of data?

Reply:
You don't have any pointer types in your struct, right? Can you make sure it passes verifier?